internal/webhook/kubernetes/projectconfig/webhook_test.go

```diff
@@ -546,6 +546,46 @@ func Test_webhook_ValidateCreate(t *testing.T) {
 					"webhook receiver name already defined at spec.webhookReceivers[0]")
 			},
 		},
+		{
+			name: "mutually exclusive webhook receiver config",
```

Comment:
We've tended toward naming test cases for the circumstances they exercise, so I might suggest:

```suggestion
			name: "webhook receiver contains config for multiple receiver types",
```